app/Http/Controllers/Admins/OrderController.php

```diff
@@ -54,7 +54,25 @@ public function edit($id)
      */
     public function update(Request $request, $id)
     {
-        //
+        Order::where('id', $id)->update(['status' => $request->status]);
+        $order = Order::findOrFail($id);
+        $orderItems = $order->orderItems;
+        if (isset($product_id)) {
```

Comment:
```suggestion
        if (isset($productId)) {
```
Biến này không có trước đó mà e